integration/airflow/openlineage/airflow/extractors/sql_extractor.py

```diff
@@ -182,6 +190,26 @@ def _get_output_facets(self) -> Dict[str, BaseFacet]:
     def _normalize_name(name: str) -> str:
         return name.lower()
 
+    def attach_column_facet(self, dataset, sql_meta: SqlMeta):
+        if not len(sql_meta.column_lineage):
+            return
+        dataset.custom_facets['columnLineage'] = ColumnLineageDatasetFacet(
```

Comment:
This shouldn't be considered custom facet. I know we might want to get rid of `Dataset` class from `common` package but still it look better to have `column_lineage` field in current implementation.

Reply:
`custom_facets` is a container for... normal facets in current implementation. We can't change it because of compatibility - custom extractors could use it.